packages/components/src/TabBar/__snapshots__/TabBar.test.js.snap

```diff
@@ -0,0 +1,104 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TabBar component should render 1`] = `
+<Uncontrolled(TabContainer)
+  activeKey="3"
+  generateChildId={undefined}
+  id="my-tabs"
+  onKeyDown={[Function]}
+  onSelect={[Function]}
+>
+  <div>
+    <Nav
+      bsClass="nav"
+      bsStyle="tabs"
+      className="tc-tab-bar"
+      justified={false}
+      pullLeft={false}
+      pullRight={false}
+      stacked={false}
+    >
+      <NavItem
+        active={false}
+        componentClass="button"
+        data-feature="action.1"
+        disabled={false}
+        eventKey="1"
+        label="Tab1"
+      >
+        Tab1
+      </NavItem>
+      <NavItem
+        active={false}
+        componentClass="button"
+        data-feature="action.2"
+        disabled={false}
+        eventKey="2"
+        label="Tab2"
+      >
+        Tab2
+      </NavItem>
+      <NavItem
+        active={false}
```

Comment:
Because we are un shallow rendering. The Nav and NavItems are not rendered. The active is resolved at rendering where Nav clone the NavItems passing extra props, including active.

If I switch to mount, the snapshot is reaaally hard to read.
So let's keep it like that, we test that we use the right react-bootstrap markup, and we don't test react-bootstrap behavior